actors/README.md

```diff
@@ -173,8 +174,7 @@ textField.onKeyboardFocusEvent { focusEvent, actor ->
 }
 ```
 
-Chaining actions with infix `then` function (`SequenceAction` utility):
-
+Chaining actions with infix `then` function (`SequenceAction` utility) and infix `along` function
```

Comment:
Since you haven't updated the example, revert this line. `along`/`/` usage examples would be welcome, although not necessary.

Reply:
I went ahead and added examples.

Reply:
I added an example.